<ui version="4.0" >
 <class>KWPageLayout</class>
 <widget class="QWidget" name="KWPageLayout" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>332</width>
    <height>324</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="3" column="4" >
    <widget class="QRadioButton" name="landscape" >
     <property name="text" >
      <string>Landscape</string>
     </property>
    </widget>
   </item>
   <item row="4" column="4" >
    <widget class="QRadioButton" name="facingPages" >
     <property name="text" >
      <string>Facing pages</string>
     </property>
    </widget>
   </item>
   <item rowspan="5" row="1" column="5" >
    <widget class="QWidget" native="1" name="previewPane" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>5</hsizetype>
       <vsizetype>5</vsizetype>
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="5" >
    <widget class="QGroupBox" name="margins" >
     <property name="title" >
      <string>Margins</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item rowspan="4" row="0" column="2" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType" >
         <enum>QSizePolicy::MinimumExpanding</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>10</width>
          <height>10</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="rightLabel" >
        <property name="text" >
         <string>Page Edge:</string>
        </property>
        <property name="buddy" >
         <cstring>pageEdgeMargin</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="leftLabel" >
        <property name="text" >
         <string>Binding Edge:</string>
        </property>
        <property name="buddy" >
         <cstring>bindingEdgeMargin</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="0" >
       <widget class="QLabel" name="label_12" >
        <property name="text" >
         <string>Bottom:</string>
        </property>
        <property name="buddy" >
         <cstring>bottomMargin</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="KoUnitDoubleSpinBox" name="pageEdgeMargin" />
      </item>
      <item row="0" column="0" >
       <widget class="QLabel" name="label_9" >
        <property name="text" >
         <string>Top:</string>
        </property>
        <property name="buddy" >
         <cstring>topMargin</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="KoUnitDoubleSpinBox" name="bindingEdgeMargin" />
      </item>
      <item row="0" column="1" >
       <widget class="KoUnitDoubleSpinBox" name="topMargin" />
      </item>
      <item row="3" column="1" >
       <widget class="KoUnitDoubleSpinBox" name="bottomMargin" />
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Unit:</string>
     </property>
     <property name="buddy" >
      <cstring>units</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="label_2" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>0</hsizetype>
       <vsizetype>5</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>Size:</string>
     </property>
     <property name="buddy" >
      <cstring>sizes</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="label_3" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>0</hsizetype>
       <vsizetype>5</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>Width:</string>
     </property>
     <property name="buddy" >
      <cstring>width</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="3" >
    <widget class="QLabel" name="facingPagesIcon" >
     <property name="text" >
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="0" >
    <widget class="QLabel" name="label_5" >
     <property name="text" >
      <string>Orientation:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2" >
    <widget class="QRadioButton" name="portrait" >
     <property name="text" >
      <string>Portrait</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2" >
    <widget class="QRadioButton" name="singleSided" >
     <property name="text" >
      <string>Single sides</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3" >
    <widget class="QLabel" name="landscapeIcon" >
     <property name="text" >
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="4" >
    <widget class="QComboBox" name="sizes" />
   </item>
   <item row="4" column="0" >
    <widget class="QLabel" name="label_6" >
     <property name="text" >
      <string>Facing Pages:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="4" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="KoUnitDoubleSpinBox" name="width" />
     </item>
     <item>
      <widget class="QLabel" name="label_4" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>0</hsizetype>
         <vsizetype>5</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>Height:</string>
       </property>
       <property name="buddy" >
        <cstring>height</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KoUnitDoubleSpinBox" name="height" />
     </item>
    </layout>
   </item>
   <item row="3" column="1" >
    <widget class="QLabel" name="portraitIcon" >
     <property name="text" >
      <string/>
     </property>
    </widget>
   </item>
   <item row="4" column="1" >
    <widget class="QLabel" name="singleSidedIcon" >
     <property name="text" >
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="4" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QComboBox" name="units" />
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>10</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KoUnitDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>KoUnitWidgets.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>units</tabstop>
  <tabstop>sizes</tabstop>
  <tabstop>width</tabstop>
  <tabstop>height</tabstop>
  <tabstop>portrait</tabstop>
  <tabstop>landscape</tabstop>
  <tabstop>singleSided</tabstop>
  <tabstop>facingPages</tabstop>
  <tabstop>topMargin</tabstop>
  <tabstop>bindingEdgeMargin</tabstop>
  <tabstop>pageEdgeMargin</tabstop>
  <tabstop>bottomMargin</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
